storm-server/src/main/java/org/apache/storm/daemon/supervisor/Slot.java

```diff
@@ -397,22 +403,26 @@ private static DynamicState filterChangingBlobsFor(DynamicState dynamicState, fi
     }
 
     /**
-     * State Transitions for WAITING_FOR_BLOB_LOCALIZATION state.
+     * State Transitions for WAITING_FOR_BLOB_LOCALIZATION state. This is when slot is waiting for
+     * the blob to download before the container is launched/relaunched for the first time for an assignment.
+     * for the first time and
```

Comment:
nit: this small fragment after the end of the sentence does not totally make since.